sequencing/ATACSeq/jobSubmission/6_batchRunGenotypeCheck.sh

```diff
@@ -0,0 +1,157 @@
+## - The following variables specified in config file: META_DIR, MAIN_DIR, LOG_DIR, ALIGNED_DIR, SCRIPTS_DIR, PROJECT,PEAK_DIR ||
+## - Version/directory of the following modules should be specified in config file: PICARDVERS, RVERS, SAMTVERS                ||
+## - Softwares: Picard, GATK, SAMtools, R, Pandoc,                                                                             ||
+## - For modules or references required, please refer to each subscript run in this script.                                    ||
+## - Subscripts to be in ${SUB_SCRIPTS_DIR} = ./subscripts                                                                     ||
+## - Subscripts: compareBamWithGenotypes.sh, searchBestGenoMatch.sh                                                            ||
+## - R subscripts to be in ${RSCRIPTS_DIR} = ./Rscripts                                                                        ||
+## - R scripts: collateSampleChecks.Rmd                                                                                        ||
+##                                                                                                                             ||
+## ============================================================================================================================##
+
+## ============ ##
+##    SET UP    ##
+## ============ ##
+
+## print start date and time
+echo Job started on:
+date -u
+
+## load config file provided on command line related to the specified project
+source "${1}/config.txt"
+echo "Loading config file for project: " ${PROJECT}
+echo "Project directory is: " $MAIN_DIR
+echo 'Script is running from directory: ' ${SCRIPTS_DIR}
+
+LOG_DIR=${LOG_DIR}/${USER}/${SLURM_ARRAY_JOB_ID}
+echo "Log files will be moved to dir: " $LOG_DIR
+mkdir -p $LOG_DIR
+mv ATACGenoConcorS6-${SLURM_ARRAY_JOB_ID}_${SLURM_ARRAY_TASK_ID}* $LOG_DIR
+
+## ================ ##
+##    VARIABLES     ##
+## ================ ##
+
+if [[ $2 == '' ]] 
+then
+  echo "No step specified. All steps in the script will be run. "
+fi
+
+##check array specified and exit if not
+if [[ ${SLURM_ARRAY_TASK_ID} == '' ]]
+then 
+    { echo "Job does not appear to be an array. Please specify --array on the command line." ; exit 1; }
+fi
+
+## check step method matches required options and exit if not
+if [[ ! $2 =~ "COMPARE" ]] && [[ ! $2 =~ "GENCHECK" ]] && [[ ! $2 =~ "SWITCH" ]] &&[[ ! $2 == '' ]];
+then 
+    { echo "Unknown step specified. Please use COMPARE, GENCHECK, SWITCH or some combination of this as a single string (i.e. SPLIT,CHECK)" ; exit 1; }            
+fi
+
+## ============ ##
+##    STEPS     ##
+## ============ ##
+
+## option COMPARE: genotype of samples are verified by comparing them with existing genotype information   
+if [ $# = 1 ] || [[ $2 =~ 'COMPARE' ]]
+then
+  module purge
+  module load $PICARDVERS
+  module load GATK
```

Comment:
Is a specific version of GATK required? I don't really know what this is, but all other modules loaded have a specific version requirement in the config file. Why not this one?